lib/rules/jsx-uses-react.js

```diff
@@ -33,12 +33,4 @@ module.exports = function(context) {
 
 };
 
-module.exports.schema = [{
-  type: 'object',
-  properties: {
-    pragma: {
-      type: 'string'
-    }
-  },
-  additionalProperties: false
-}];
+module.exports.schema = [];
```

Comment:
I just tried this out with a different rule that has a schema of `[]` and it produced an error like:

```
Configuration for rule "plugin/rule" is invalid:
Value "config value" has more items than allowed.
```